docs/docs/how-to/previews-deploys-hosting/adapters.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Adapters
+---
+
+## Introduction
+
+Adapters are responsible for taking the production output from Gatsby and turning it into something your deployment platform understands. They make it easier to build and deploy Gatsby on any deployment platform.
+
+Gatsby has different [rendering options](/docs/conceptual/rendering-options/) and features like DSG and SSR require more setup than classic SSG. Users can also set [HTTP headers](/docs/how-to/previews-deploys-hosting/headers/) or create [redirects](/docs/reference/config-files/actions/#createRedirect). Gatsby passes all the required information during the build to its adapters so that they can _adapt_ these outputs for deployment.
+
+This feature was added in `gatsby@5.X.0`.
+
```

Comment:
I found the list you added for the `adapt` hook (what the hook should do) really clearly explains what an adapter does. I wonder if it might be helpful to include that here too as part of the introduction.

It could be a separate section before "Finding" or could just merge with the paragraph you have above.

Here's a suggestion as a separate section:

```suggestion
## What an adapter does

An adapter takes the output from Gatsby build and prepares it for deployment on a specific platform. Here are some of the actions an adapter automatically takes:

- Applies HTTP headers to assets
- Applies redirects and rewrites. The adapter can also create its own redirects or rewrites if necessary, for example to map serverless functions to internal URLs.
- Wraps serverless functions coming from Gatsby with platform-specific code (if necessary). Gatsby will produce [Express](https://expressjs.com/)-like handlers.
- Possibly uploads assets to CDN
```

Here's a suggestion where we merge with the paragraph above:

```suggestion

Gatsby has different [rendering options](/docs/conceptual/rendering-options/) and features like Deferred Static Generation (DSG) and Server-Side rendering (SSR) require more setup than classic static site generation (SSG). Users can also set [HTTP headers](/docs/how-to/previews-deploys-hosting/headers/) or create [redirects](/docs/reference/config-files/actions/#createRedirect). 

Gatsby passes all the required information during the build to adapters to prepare these outputs for deployment on a specific platform. Here are some of the actions an adapter automatically takes:

- Applies HTTP headers to assets
- Applies redirects and rewrites. The adapter can also create its own redirects or rewrites if necessary, for example to map serverless functions to internal URLs.
- Wraps serverless functions coming from Gatsby with platform-specific code (if necessary). Gatsby will produce [Express](https://expressjs.com/)-like handlers.
- Possibly uploads assets to CDN

```